src/listen/websocket.rs

```diff
@@ -65,15 +70,12 @@ struct FileChunker {
 
 impl Transcription<'_> {
     pub fn stream_request(&self) -> StreamRequestBuilder<'_> {
-        self.stream_request_with_options(None)
+        self.stream_request_with_options(Options::builder().build())
     }
 
-    pub fn stream_request_with_options<'a>(
-        &'a self,
-        options: Option<&'a Options>,
-    ) -> StreamRequestBuilder<'a> {
+    pub fn stream_request_with_options(&self, options: Options) -> StreamRequestBuilder<'_> {
```

Comment:
Callout: I changed this public interface to take a `crate::common::options::Options` instead of an `Option<&crate::common::options::Options>`.  It says it takes options, so only sometimes taking them is weird, especially if we also provide a version that doesn't take them, and since we can get the same behavior by passing an empty `Options`.

Reply:
Just checked:  This function was not available in 0.5.0, so this isn't a breaking change.